Jenkinsfile

```diff
@@ -28,7 +28,7 @@ for (int i = 0; i < platforms.size(); ++i) {
 
                 stage('Archive') {
                     /* Archive the test results */
-                    junit '**/target/surefire-reports/TEST-*.xml'
+                    // junit '**/target/surefire-reports/TEST-*.xml'
```

Comment:
could we just use `allowEmptyResults` so that this is not commented and still not failing if there is no test?